lavaplay/ws.py

```diff
@@ -38,22 +36,18 @@ def __init__(
         self._headers = {
             "Authorization": password,
             "User-Id": str(user_id),
-            "Client-Name": f"Lavaplay.py/{__version__}",
-            "Num-Shards": str(shards_count)
+            "Client-Name": f"Lavaplay.py/{__version__}"
         }
         self._loop = loop or node.loop
         self.emitter: Emitter = node.event_manager
         self.is_connect: bool = False
-        self.resume_key = resume_key or self.node._resume_key or generate_resume_key()
```

Comment:
The same point, Resume key is feature

Reply:
https://lavalink.dev/api/rest#update-session